### PR TITLE
Event times

### PR DESCRIPTION
**Pull request by Raemon**
(includes the LWTooltip PR)

### CODE REVIEW

File: packages/lesswrong/components/localGroups/EventTime.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ function getDateFormat(dense, isThisYear) {
       return 'DD MMM YYYY'; // 5 Jan 2020
     }
   } else {
-    return 'Do MMMM YYYY'; // 5th January 2020
+    return 'dddd Do MMMM YYYY'; // 5th January 2020
```
Comment (Discordius) [inline packages/lesswrong/components/localGroups/EventTime.tsx]: Update the comment

File: packages/lesswrong/components/localGroups/EventTime.tsx [hunk 2]
```diff
@@ -26,14 +26,14 @@ const EventTime = ({post, dense=false}: {
   const isThisYear = moment(new Date()).format("YYYY") === moment(start).format("YYYY");
 
   // Date and time formats
-  const timeFormat = 'h:mm A z'; // 11:30 AM PDT
+  const timeFormat = 'h:mm a z'; // 11:30 AM PDT
```
Comment (Discordius) [inline packages/lesswrong/components/localGroups/EventTime.tsx]: Update the comment

File: packages/lesswrong/components/localGroups/EventTime.tsx [hunk 3]
```diff
@@ -26,14 +26,14 @@ const EventTime = ({post, dense=false}: {
...
   const dateFormat = getDateFormat(dense, isThisYear);
   const dateAndTimeFormat = dateFormat+', '+timeFormat;
   const calendarFormat = {sameElse : dateAndTimeFormat}
 
   // Alternate formats omitting the timezone, for the start time in a
   // start-end range.
-  const startTimeFormat = 'h:mm A'; // 11:30 AM
+  const startTimeFormat = 'h:mm a'; // 11:30 AM
```
Comment (Discordius) [inline packages/lesswrong/components/localGroups/EventTime.tsx]: Update comment

**Comment by jimrandomh** [comment]
Based off this PR: https://github.com/LessWrong2/Lesswrong2/pull/2933

**Comment by Raemon** [comment]
Needs @Discordius to approve

**Pull request merged**